CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ## Unreleased
 
+### Fixed
+
+- Fixed DNA Tester now actually adding new samples
+- TraitorButton config files should now actually work
+- Entities on roundstart are now triggered with the actual value of ROUND_ACTIVE
```

Comment:
It's not about general map entities, the entity in question is the `ttt_map_settings` entity